examples/asr/asr_vad/speech_to_text_with_frame_vad.py

```diff
@@ -0,0 +1,634 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
can't we merge the two types of vad to one script?

Reply:
merged into one script